website/project/views/node.py

```diff
@@ -251,9 +251,10 @@ def project_before_template(auth, node, **kwargs):
 @must_be_valid_project
 @must_be_contributor_or_public_but_not_anonymized
 @must_not_be_registration
-@ember_flag_is_active(features.EMBER_PROJECT_REGISTRATIONS)
 def node_registrations(auth, node, **kwargs):
-    return _view_project(node, auth, primary=True, embed_registrations=True)
+    if request.path.startswith('/project/'):
```

Comment:
I'm not sure I fully understand this change and why it's necessary. Is this to preserve the v1 route?

Reply:
Ah, found the similar changes from previous reviews: https://github.com/CenterForOpenScience/osf.io/pull/8780#issuecomment-434786991
https://github.com/CenterForOpenScience/osf.io/pull/8781#issuecomment-435452350 👍